class JobNotFoundException(Exception):
    pass


class JobAlreadyRunningException(Exception):
    pass
